cli/compose/convert/service.go

```diff
@@ -217,13 +217,17 @@ func convertServiceSecrets(
 		if gid == "" {
 			gid = "0"
 		}
+		mode := secret.Mode
+		if mode == 0 {
```

Comment:
@vdemeester do you have another way to do this that doesn't actively prevent the user from setting mode 0 on purpose? Principle of least surprise?